extensions/hooks/_hook_copy_pdbs_to_package.py

```diff
@@ -5,6 +5,7 @@
 import re
 from io import StringIO
 from conan.errors import ConanException
+from conans.util.env import get_env
```

Comment:
Not sure of adding this import from `conans`. Maybe it should be migrated to `conan` module first?